api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -234,6 +234,18 @@ message RedisProxy {
   // client. If an AUTH command is received when the password is not set, then an "ERR Client sent
   // AUTH, but no password is set" error will be returned.
   config.core.v3.DataSource downstream_auth_password = 6 [(udpa.annotations.sensitive) = true];
+
+  // If a username is provided an ACL style AUTH command will be required with a username and password.
+  // Authenticate Redis client connections locally by forcing downstream clients to issue a `Redis
+  // AUTH command <https://redis.io/commands/auth>`_ with this username and the downstream_auth_password
```

Comment:
Nit: for fields such as `downstream_auth_password`, either add a `:ref:` or write as `*downstream_auth_password*` if they are proximal (which is the case here).

Reply:
Fixed.